docs/usage.md

```diff
@@ -113,14 +113,17 @@ To manually add `debug` to `__builtins__`, add the following to `sitecustomize.p
 which is always imported.
 
 ```py
-# add devtools `debug` function to builtins
-import builtins
-try:
-    from devtools import debug
-except ImportError:
-    pass
-else:
-    setattr(builtins, 'debug', debug)
+import sys
+# we don't install here for pytest as it breaks pytest, it is
+# installed later by a pytest fixture
+if not sys.argv[0].endswith('pytest'):
```

Comment:
Command name might also be `py.test`

It's an old name and hence unlikely, but is still included when installing `pytest` so can be used.

Reply:
This copies from the script we auto-install, I think this is a good start.